snap/snapcraft.yaml

```diff
@@ -44,6 +44,10 @@ parts:
         - patchelf
         - squashfs-tools
         - xdelta3
+    organize:
+        # Put snapcraftctl into its own directory that can be included in the PATH
+        # without including other binaries.
+        bin/snapcraftctl: bin/snapcraft-utils/snapcraftctl
```

Comment:
can we name this "internal-bin" at the top level instead? so it looks like,
    bin/snapcraftctl: internal-bin/snapcraftctl

Is this in anticipation of more tooling?

Reply:
> can we name this "internal-bin" at the top level instead?

Definitely, no opinion on this whatsoever, as long as it's a directory that only contains tools _intended_ for use in scriptlets. Oh... actually, maybe it should be called "scriptlet-bin" then, since it's actually more of a public bin within a private than a private within a public (if that makes sense).

> Is this in anticipation of more tooling?

No, I don't see any more snapcraftctl-like things in the future, but adding e.g. `$SNAP/bin` to the path would have exposed all sorts of internal things to scriptlets.

Reply:
scriptlet-bin is fine, I wasn't advocating for internal-bin in any way :-)

Also, not advocating for $SNAP/bin either, it has to be a nice subset of things we want to expose.